lib/msf/core/payload/windows/rc4.rb

```diff
@@ -0,0 +1,115 @@
+# -*- coding: binary -*-
+
+require 'msf/core'
+
+module Msf
+
+###
+#
+# RC4 decryption stub for Windows ARCH_X86 payloads
+#
+###
+
+module Payload::Windows::Rc4
+
+  #
+  # Register rc4 specific options
+  #
+  def initialize(*args)
+    super
+    register_advanced_options([ OptString.new("RC4PASSWORD", [true, "Password to derive RC4 key from"]) ], self.class)
```

Comment:
RC4PASSWORD is now an advanced option, when it used to be a regular option. I recommend to leave it as regular option, since it is a required one.